### PR TITLE
WalletConnect Mobile Login Hotfix

### PR DESCRIPTION
**Pull request by alexyoung23j**
## Link to Issue
Closes: #3327 

## Description of Changes
- One liner fix for mobile login bug reported by 1inch and others. 

## Test Plan
- Tested locally- this is a bit difficult to test but my setup was to use ngrok to create a url being served by my localhost that I could access on my phone. 

## Deployment Plan

1. 

## Other Considerations

-

### CODE REVIEW

**Comment by jnaviask** [comment]
Weird race conditions on frack. Hard to follow UI for connect + sign, at least on trust wallet. What wallets should I test with?

**Comment by jnaviask** [comment]
Was able to login using rainbow wallet, although had to manually navigate back to the wallet after connecting in order to sign the message.

**Review by CowMuon — APPROVED** [review]
Jankyish, but shippable. Fast follow cleanup here? @jnaviask @alexyoung23j
